src/main/java/com/uci/orchestrator/Consumer/ReactiveConsumer.java

```diff
@@ -43,11 +40,7 @@
 import javax.xml.bind.JAXBException;
 import java.io.ByteArrayInputStream;
 import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Same with this import too, specific classes are preferable.

Reply:
Yes, you are right but when we use more than 5 classes from same package IDE automatically do this. I will try to figure out next time.